Standards/WCAG2AAA/Sniffs/Principle4/Guideline4_1/4_1_2.js

```diff
@@ -199,7 +199,7 @@ _global.HTMLCS_WCAG2AAA_Sniffs_Principle4_Guideline4_1_4_1_2 = {
             var requiredValue = requiredValues[nodeName];
 
             // Any element that doesn't have specific handling must have content.
-            if (!matchingRequiredNames) {
+            if (!matchingRequiredNames && nodeName !== 'input_hidden') {
```

Comment:
If I understand correctly, that array should only enumerate the required attributes of a given input type, whereas input_hidden actually shouldn't have any requirements for labelling. I tried that option before settling on this PR using `input_hidden: [],` and `input_hidden: [''],` but that still flags the following issue:

> [HTMLCS] ERROR|WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputHidden.Name|input||This hidden input element does not have a name available to an accessibility API. Valid names are: .|<input type="hidden" name="option_list" value="" class="js-option-list">